redis/rediscache.go

```diff
@@ -0,0 +1,39 @@
+package rediscache
```

Comment:
I would like to name it redis but I am afraid of a name collision with https://github.com/go-redis/redis